lib/miq_automation_engine/engine/miq_ae_object.rb

```diff
@@ -116,7 +116,7 @@ def initialize(workspace, ns, klass, instance, object_name = nil)
 
       Benchmark.realtime_block(:inherit_time) do
         # Who do we inherit from
-        @inherits = @aec.inherits.blank? ? BASE_OBJECT : @aec.inherits
+        @inherits = BASE_OBJECT
```

Comment:
@billfitzgerald0120 
The tests are failing because we still have a spec that is testing the inheritance which needs to be removed. That test is expecting the inheritance to work.